scripts/protocgen.sh

```diff
@@ -2,8 +2,35 @@
 
 set -eo pipefail
 
+git clone https://github.com/tendermint/spec.git
+
+# Edit this line to clone your branch, if you are modifying protobuf files
+cd spec && git checkout v0.7.1 && cd ..
```

Comment:
Another possible way to do this without having to clone the whole repo would be to fetch the tag's archive, and only extract the proto directory, e.g.,

```sh
VERS=0.7.1
curl -qL "https://github.com/tendermint/spec/archive/refs/tags/v${VERS}.tar.gz" | tar -xjf - spec-"$VERS"/proto/
mv spec-"$VERS" spec
```

YMMV.

Reply:
oh awesome, ill implement that. Thanks for the pointer!!